install/linux/usr/share/odemis/sim/enzel-sim.odm.yaml

```diff
@@ -1,416 +1,414 @@
 ENZEL-Sim: {
-    class: Microscope,
-    role: cryo-secom,
-    children: [
-        "Sample Thermostat", "Cooler Controller",
-        "SEM E-beam Full", "SEM Detector",
-        "EBeam Focus",
-        "Light Source", "Filter Wheel", "Stigmator",
-        "Camera", "Optical Objective",
-        "Objective Aligner", "Optical Focus",
-        "Linked Stage",
-    ],
+  class: Microscope,
+  role: cryo-secom,
+  children: [
+      "Sample Thermostat", "Cooler Controller",
+      "SEM E-beam Full", "SEM Detector",
+      "EBeam Focus",
+      "Light Source", "Filter Wheel", "Stigmator",
+      "Camera", "Optical Objective",
+      "Objective Aligner", "Optical Focus",
+  ],
 }
 
 
 # Lakeshore 335, which controls the temperature
 "Sample Thermostat": {
-    class: lakeshore.Lakeshore,
-    role: sample-thermostat,
-    init: {
-#        port: "/dev/ttyUSB*",
-        port: "/dev/fake",  # For simulator
-        sensor_input: "B",
-        output_channel: 2,
-    },
-    properties: {
-        targetTemperature: -173, # °C
-    },
-    metadata: {
-        # TODO: Change values from test to real values
-        SAFE_REL_RANGE: [-5, 5],  # °C, safe operating range relative to targetTemperature
-        SAFE_SPEED_RANGE: [-0.5, 0.5] # °C/s, safe operating speed
-    },
+  class: lakeshore.Lakeshore,
+  role: sample-thermostat,
+  init: {
+    #        port: "/dev/ttyUSB*",
+    port: "/dev/fake",  # For simulator
+    sensor_input: "B",
+    output_channel: 2,
+  },
+  properties: {
+    targetTemperature: -173, # °C
+  },
+  metadata: {
+    # TODO: Change values from test to real values
+    SAFE_REL_RANGE: [ -5, 5 ],  # °C, safe operating range relative to targetTemperature
+    SAFE_SPEED_RANGE: [ -0.5, 0.5 ] # °C/s, safe operating speed
+  },
```

Comment:
Because of the style/tab changes made through this file it is hard to review. It would be nice if that is a seperate PR>
That said I'm in favor of these changes and the file does look ok to me.